examples/gym/configs/config_example_map.yaml

```diff
@@ -4,23 +4,14 @@ run_name: 'map_wide'
 perf_num: 6
```

Comment:
I still have problem if to keep singular or plural in directory names :(. `osgar/confg` but `osgar/tools` ... well, ignore